.github/workflows/docs-publish.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and Deploy Chain docs
+
+on:
+  push:
+    branches:
+      - develop
```

Comment:
So now all versions will override each other since everything is deployed on the same branch? Could we perhaps name the branch differently depending on what we are pushing? Or just remove the `master` deployment and only have `develop` and the latest release, which are also the only two docs we would care about.

Reply:
Removed `master`

Reply:
Removed `master`

Reply:
Should we still name the deployment branches different between `develop` and releases?

Reply:
I dynamically generate the deployment branch `gh-pages-${{ github.ref_name }}`, I hope that is not an issue for the linking @ChrisChinchilla

Reply:
@ggera Dunno, I guess we'll see